src/messages/discord_embeds.py

```diff
@@ -71,8 +71,7 @@ def send_gold(self):
     def send_message(self, msg: str):
         msg = f"{Config().general['name']}: {msg}"
         e = discord.Embed(title=f"Update:", description=f"```{msg}```", color=Color.dark_teal())
-        if not Config().general['discord_status_condensed']:
-            e.set_thumbnail(url=f"{self._psnURL}36L4a4994.png")
+        e.set_thumbnail(url=f"{self._psnURL}36L4a4994.png")
```

Comment:
Don't want the image in the condensed version.  It was only adding the image if it was not condensed

Remove this line


```suggestion

```

Reply:
fixed